.changeset/spicy-donkeys-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': patch
```

Comment:
I think you can safely remove this changeset since it's not impacting anything library related! 👍